tests/visual/treeview.html

```diff
@@ -0,0 +1,173 @@
+<!doctype html>
+<html lang="en">
+<head>
+    <title>Treeview</title>
+
+    <link rel="stylesheet" href="../../packages/default/dist/all.css" />
+    <link rel="stylesheet" href="assets/styles.css" />
+    <script src="assets/scripts.js"></script>
+
+    <style>
+        #test-area {
+            max-width: 200px;
```

Comment:
This crops the screenshot area, so it should probably be deleted.

![image](https://user-images.githubusercontent.com/90405/39426806-eaf79d6a-4c89-11e8-8b3d-849218a025f8.png)